### PR TITLE
[TT-4713] Update Ubuntu SQL installation instructions

### PR DESCRIPTION
**Pull request by jeffy-mathew**

### CODE REVIEW

File: tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/dashboard.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ This tutorial has been tested on Ubuntu 16.04 & 18.04 with few if any modificati
 
 ### Prerequisites
 
-- Have MongoDb and Redis installed - see [here][2] for details.
+- Have MongoDb/SQL and Redis installed - see [here][2] for details.
```
Comment (furkansenharputlu) [inline tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/dashboard.md]: can we change this one to MongoDB?

**Review by marksou — APPROVED** [review]
👍

**Comment by marksou** [comment]
release to release-4-lts

**Pull request merged**